toolbox/gui/panel_scout.m

```diff
@@ -4051,12 +4051,13 @@ function PlotScouts(iScouts, hFigSel)
         return;
     end
     % Get anatomy file
-    [sSubject, iSubject] = bst_get('SurfaceFile', sSurf.FileName);
+    [sSubject, iSubject, iSurface] = db_get('SurfaceFile', sSurf.FileName);

```

Comment:
Error: Output argument "varargout{3}" (and maybe others) not assigned during call to "db_get".

Reply:
Fixed in 1e444577ea3f374715cb0697526a0880e4a3ed63